web/src/lib/utils/file-uploader.ts

```diff
@@ -17,16 +17,32 @@ const getExtensions = async () => {
   }
   return _extensions;
 };
+// the following set of interfaces ensure that either albumId OR assetId
+// may be specified, but not both.
+interface FileUploadParams {
+  multiple?: boolean;
+}
+interface FileUploadWithAlbum extends FileUploadParams {
+  albumId?: string;
+  assetId?: never;
```

Comment:
Why is that type `never`?

Reply:
I think this makes it possible to access item.assetId when it is this type of above without needing to cast it.